frontend/packages/local-storage-operator-plugin/integration-tests/views/disks-list-page.views.ts

```diff
@@ -0,0 +1,30 @@
+import { $, $$ } from 'protractor';
+import { resourceRows } from '@console/internal-integration-tests/views/crud.view';
+import { click } from '@console/shared/src/test-utils/utils';
+
+export const page = {
+  diskTab: $('a[data-test-id="horizontal-link-Disks"]'),
+  resourceTitles: $$('[data-test-id="resource-title"]'),
+};
+
+export const diskList = {
+  headers: ['Name', 'Disk State', 'Type', 'Model', 'Capacity', 'Filesystem'],
+  dropdownFilter: $('button[data-test-id="filter-dropdown-toggle"]'),
+  dropdownFilterHeader: $('.pf-c-dropdown__group-title'),
+  textFilter: $('input[data-test-id="item-filter"]'),
+};
+
+export const clickFilterDropdown = () => diskList.dropdownFilter.click();
```

Comment:
This function can go inside `diskList` object as well.

Reply:
Hey,
The properties could run into self-referencing errors on moving inside object.

Reply:
Its not harm keeping outside too.